client/components/tinymce/plugins/insert-menu/plugin.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import tinymce from 'tinymce/tinymce';
+import { renderToString } from 'react-dom/server';
+
+import Gridicon from 'components/gridicon';
+
+import menuItems from './menu-items';
+
+const initialize = editor => {
+	menuItems.forEach( item =>
+		editor.addMenuItem( item.name, {
+			classes: 'wpcom-insert-menu__menu-item',
+			cmd: item.cmd,
+			onPostRender() {
+				this.innerHtml( renderToString( item.item ) );
+			}
+		} )
+	);
+
+	editor.addButton( 'wpcom_insert_menu', {
+		type: 'splitbutton',
+		title: 'Insert content',
+		classes: 'btn wpcom-insert-menu insert-menu',
```

Comment:
@drw158 I would appreciate your direction here. When I cut out the `insert-menu` class the dropdown button breaks.